File: This_App_Backend/src/main/java/com/example/This_App_Backend/dto/StoresDTO/StoreDTO.java
package com.example.This_App_Backend.dto.StoresDTO;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class StoreDTO {

    private Long storeId;
    private String storeName;
    private String storeDescription;
    private String storeAddress;
    private String storeEmail;
    private String storePhoneNumber;
    private String storeBusinessHours;
    private String storeLogo;
    private Long ownerId;

}

File: this-app/src/app/models/store-admin-models/store-admin/StoreDTO.ts
export interface StoreDTO {
  storeId?: number;
  storeName: string;
  storeDescription: string;
  storeAddress: string;
  storeEmail: string;
  storePhoneNumber: string;
  storeBusinessHours: string;
  storeLogo?: string;
  ownerId?: number;  // ← Fixed typo
}